Description: disable code which is using resources not in DFSG tarball
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: not-needed
Last-Update: 2017-07-16

Index: qt3d-everywhere-src-5.10.0-beta/examples/qt3d/exampleresources/obj.qrc
===================================================================
--- qt3d-everywhere-src-5.10.0-beta.orig/examples/qt3d/exampleresources/obj.qrc
+++ qt3d-everywhere-src-5.10.0-beta/examples/qt3d/exampleresources/obj.qrc
@@ -1,8 +1,5 @@
 <RCC>
     <qresource prefix="/">
-        <file>assets/textures/pattern_09/specular.webp</file>
-        <file>assets/textures/pattern_09/normal.webp</file>
-        <file>assets/textures/pattern_09/diffuse.webp</file>
         <file>assets/obj/trefoil.obj</file>
         <file>assets/obj/toyplane.obj</file>
         <file>assets/obj/ball.obj</file>
Index: qt3d-everywhere-src-5.10.0-beta/examples/qt3d/qt3d.pro
===================================================================
--- qt3d-everywhere-src-5.10.0-beta.orig/examples/qt3d/qt3d.pro
+++ qt3d-everywhere-src-5.10.0-beta/examples/qt3d/qt3d.pro
@@ -6,15 +6,12 @@ SUBDIRS += \
     multiviewport \
     wireframe \
     shadow-map-qml \
-    materials \
     wave \
-    materials-cpp \
     scene3d \
     controls \
     anaglyph-rendering \
     planets-qml \
     instanced-arrays-qml \
-    lights \
     compute-particles \
     3d-text \
     qardboard \
@@ -23,8 +20,6 @@ SUBDIRS += \
     scene2d \
     phong-cubes
 
-qtHaveModule(multimedia): SUBDIRS += audio-visualizer-qml
-
 # qmake seems to break in some CI configurations, disable this for now
 #SUBDIRS += qgltf
 
Index: qt3d-everywhere-src-5.10.0-beta/tests/tests.pro
===================================================================
--- qt3d-everywhere-src-5.10.0-beta.orig/tests/tests.pro
+++ qt3d-everywhere-src-5.10.0-beta/tests/tests.pro
@@ -4,7 +4,6 @@ QT_FOR_CONFIG += 3dcore
 
 !package {
     SUBDIRS += auto
-    qtConfig(qt3d-extras): SUBDIRS += manual
 }
 
 # Benchmarks make sense in release mode only.
